doc/api/synopsis.md

```diff
@@ -22,23 +75,31 @@ const port = 3000;
 const server = http.createServer((req, res) => {
   res.statusCode = 200;
   res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
+  res.end('Hello World!\n');
 });
 
 server.listen(port, hostname, () => {
   console.log(`Server running at http://${hostname}:${port}/`);
 });
 ```
+Save the file, go back to the terminal window enter the following command:
```

Comment:
Nit: As above: a blank line before and after code blocks is always good. There are more of these cases below.